src/providers.rs

```diff
@@ -1,12 +1,19 @@
 use crate::*;
 
-pub const CLOUDFLARE_HOSTNAME: &str = "cloudflare-eth.com";
 pub const ANKR_HOSTNAME: &str = "rpc.ankr.com";
-pub const PUBLICNODE_ETH_MAINNET_HOSTNAME: &str = "ethereum.publicnode.com";
+pub const ALCHEMY_ETH_MAINNET_HOSTNAME: &str = "eth-mainnet.g.alchemy.com";
+pub const ALCHEMY_ETH_SEPOLIA_HOSTNAME: &str = "eth-sepolia.g.alchemy.com";
+pub const CLOUDFLARE_HOSTNAME: &str = "cloudflare-eth.com";
 pub const BLOCKPI_ETH_MAINNET_HOSTNAME: &str = "ethereum.blockpi.network";
-pub const ETH_SEPOLIA_HOSTNAME: &str = "rpc.sepolia.org";
 pub const BLOCKPI_ETH_SEPOLIA_HOSTNAME: &str = "ethereum-sepolia.blockpi.network";
+pub const PUBLICNODE_ETH_MAINNET_HOSTNAME: &str = "ethereum.publicnode.com";
 pub const PUBLICNODE_ETH_SEPOLIA_HOSTNAME: &str = "ethereum-sepolia.publicnode.com";
+pub const ETH_SEPOLIA_HOSTNAME: &str = "rpc.sepolia.org";
+
+// Limited API credentials for local testing.
+// Use `dfx canister call evm_rpc updateProvider ...` to pass your own keys.
+pub const ALCHEMY_ETH_MAINNET_CREDENTIAL: &str = "/v2/zBxaSBUMfuH8XnA-uLIWeXfCx1T8ItkM";
```

Comment:
Note that these API keys are intended as defaults for local dev environments (separate from the production keys).